aries-site/src/pages/templates/filtering.mdx

```diff
@@ -22,7 +22,7 @@ For details on the components that make up the filtering experience, see [Data d
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringCards/mockData.js',
   ]}
   docs="https://v2.grommet.io/data"
-  figma="https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=681%3A1430"
+  figma="https://www.figma.com/design/fupkQ4sOefx3qUVoV6WfJi/Template-Library?m=auto&node-id=163-77&t=r7Bf1k6QfmCKSMhJ-1"
```

Comment:
is this the correct link this takes me to form page? @julauxen

Reply:
Hey @britt6612, yes, it's the only place we currently have filter examples. We might need to update this later to a more detailed template.

Reply:
hmm I feel like the filtering example in the form is not the right filtering. That one is filtering validation on the form I think what we are looking for is a filtering in data and friends example. Should we just remove link?

Reply:
hey @britt6612, we also have an example without validation that looks pretty close to the one on the website on that same link: 
![Screenshot 2024-07-11 at 10 21 41](https://github.com/grommet/hpe-design-system/assets/158175533/f2b99d9d-8e79-4689-a792-6fa8d1089ac3)

Reply:
aw okay yes thats fine I missed this!